eZ/Publish/Core/settings/search_engines/elasticsearch/slots.yml

```diff
@@ -0,0 +1,130 @@
+parameters:
+    ezpublish.search.elasticsearch.slot.class: eZ\Publish\Core\Search\Common\Slot
+    ezpublish.search.elasticsearch.slot.publish_version.class: eZ\Publish\Core\Search\Common\Slot\PublishVersion
+    ezpublish.search.elasticsearch.slot.copy_content.class: eZ\Publish\Core\Search\Common\Slot\CopyContent
+    ezpublish.search.elasticsearch.slot.delete_content.class: eZ\Publish\Core\Search\Common\Slot\DeleteContent
+    ezpublish.search.elasticsearch.slot.delete_version.class: eZ\Publish\Core\Search\Common\Slot\DeleteVersion
+    ezpublish.search.elasticsearch.slot.create_location.class: eZ\Publish\Core\Search\Common\Slot\CreateLocation
+    ezpublish.search.elasticsearch.slot.update_location.class: eZ\Publish\Core\Search\Common\Slot\UpdateLocation
+    ezpublish.search.elasticsearch.slot.delete_location.class: eZ\Publish\Core\Search\Common\Slot\DeleteLocation
+    ezpublish.search.elasticsearch.slot.create_user.class: eZ\Publish\Core\Search\Common\Slot\CreateUser
+    ezpublish.search.elasticsearch.slot.create_user_group.class: eZ\Publish\Core\Search\Common\Slot\CreateUserGroup
+    ezpublish.search.elasticsearch.slot.move_user_group.class: eZ\Publish\Core\Search\Common\Slot\MoveUserGroup
+    ezpublish.search.elasticsearch.slot.copy_subtree.class: eZ\Publish\Core\Search\Common\Slot\CopySubtree
+    ezpublish.search.elasticsearch.slot.move_subtree.class: eZ\Publish\Core\Search\Common\Slot\MoveSubtree
+    ezpublish.search.elasticsearch.slot.trash.class: eZ\Publish\Core\Search\Common\Slot\Trash
+    ezpublish.search.elasticsearch.slot.recover.class: eZ\Publish\Core\Search\Common\Slot\Recover
+    ezpublish.search.elasticsearch.slot.hide_location.class: eZ\Publish\Core\Search\Common\Slot\HideLocation
+    ezpublish.search.elasticsearch.slot.unhide_location.class: eZ\Publish\Core\Search\Common\Slot\UnhideLocation
+    ezpublish.search.elasticsearch.slot.set_content_state.class: eZ\Publish\Core\Search\Common\Slot\SetContentState
+
+services:
+    ezpublish.search.elasticsearch.slot:
+        class: %ezpublish.search.elasticsearch.slot.class%
+        abstract: true
+        arguments:
+            - @ezpublish.api.inner_repository
+            - @ezpublish.api.persistence_handler
+            - @ezpublish.spi.search
+
+    ezpublish.search.elasticsearch.slot.publish_version:
+        parent: ezpublish.search.elasticsearch.slot
+        class: %ezpublish.search.elasticsearch.slot.publish_version.class%
+        tags:
+            - {name: ezpublish.search.elasticsearch.slot, signal: ContentService\PublishVersionSignal}
```

Comment:
@andrerom: to be able to pass all signal slots to `SignalDispatcher` constructor in its factory, we should use full namespace for signals here. Should I go with this or attach namespace in `SignalDispatcherFactory` (as done in [`DefaultSignalDispatcher::attach`](https://github.com/alongosz/ezpublish-kernel/blob/88697520fae1de013d4fa50e0fd51c4daf43d8a4/eZ/Publish/Core/SignalSlot/SignalDispatcher/DefaultSignalDispatcher.php#L88-L92))?